### ISSUE TITLE
PHP Fatal error Configurable.php on line 33

### ISSUE THREAD

**Issue by cdalsass** [wigman/AjaxSwatches#47]
Hi Wigman,

After installing the module on Mage Community 1.9.2.2, I'm getting this error on my product pages. Do you have any idea why?
Also, I notice my Administration -> Config -> Catalog -> Configurable Swatches section is no longer present. Is that expected?

Thanks!
Charlie

**Comment by wigman** [comment]
Hi Charlie,

Can you get me a more detailed error? enable error logging and check system.log or the servers php error log files.
Please also mention which version of my plugin you're using and check if everything is working when my plugin is disabled to make sure the swatches are working properly without my plugin.

**Comment by wigman** [comment]
no response = no problem?

**Comment by cdalsass** [comment]
Sorry, after running into problems I ended up using a commercial install instead. Thanks so much for following up.

**Comment by wigman** [comment]
Hi cdalsass, would you care to share what solution you used? (curiosity)

Thanks for replying!

**Comment by cdalsass** [comment]
Now that you've pressed, me. I'm embarrassed to say that I had your module mixed up with another one I had installed recently. I am still using your module, but had sidelined it for awhile. As far as I know (from Google searches etc), there is no commercial alternative for your module at this time. I got this issue taken care of, but  still have a few quirks I might follow up later on. Thanks again for following up and great module!
